tests/test_run_service.py

```diff
@@ -710,9 +804,53 @@ def ignore_patterns(path, names):
         # Switch to temporary directory
         os.chdir(cls.temp_dir.name)
         cls.logger.info(f"Changed working directory to: {cls.temp_dir.name}")
+
+        # Handle memeooorr config modifications if needed
+        if "memeooorr" in cls.config_path.lower():
+            temp_config_path = os.path.join(cls.temp_dir.name, 'configs', os.path.basename(cls.config_path))
+            try:
+                with open(temp_config_path, 'r') as f:
+                    config_data = json.load(f)
+                
+                # Update hash
+                config_data['hash'] = "bafybeicpoldxi2xsjbczckmhdinp3x4rttiz3mtw7slpaqs5zw3pdvzamq"
```

Comment:
:')
I [meant](https://github.com/valory-xyz/quickstart/pull/24#discussion_r1923309279) to keep only the env_variables temporary. We should still keep the hash change permanent. Because if we hardcode the hash here, then all the future changes that we do in the config file hash will not be tested.